src/main/resources/i18n/ru/sbtqa/tag/apifactory/stepdefs/GenericStepDefs/en.properties

```diff
@@ -0,0 +1,4 @@
+userSendRequestNoParams=^user sends request (?:for|to|about) \\((.*?)\\)$
+userSendRequestTableParam=^user sends request (?:for|to|about) \\((.*?)\\) with parameters$
```

Comment:
For what?

`^user sends request (?:for|to|about) \\((.*?)\\) with parameters$ `
catched steps with cucumber DataTable like
```
* user sends request to (add new popular brand) with parameters
      | brandId | 777   |
      | name    | Lotus |
```
Whereas, 
`^user sends request (?:for|to|about) \\((.*?)\\) with parameters.*$`
will catch everything like
`user sends request to (add new popular brand) with parameters !@$SWFPCV<A{}}.][321]"`